exe/server/Dockerfile

```diff
@@ -1,33 +1,14 @@
 FROM golang:1.14 as builder
```

Comment:
So, greatly simplified since we don't need Rust for building, or OpenCL and other things.
Now, `CGO_ENABLED=0` can be used, and also `alpine` as the base final image for running.